docker.d/worker.yml

```diff
@@ -0,0 +1,18 @@
+work_dir: { "$eval": "WORK_DIR" }
+mark_as_shipped_schema_file: { "$eval": "MARK_AS_SHIPPED_SCHEMA_FILE" }
+taskcluster_scope_prefix: { "$eval": "TASKCLUSTER_SCOPE_PREFIX" }
+verbose: { "$eval": "VERBOSE == 'true'" }
+ship_it_instances:
+  $if: 'ENV == "prod"'
```

Comment:
@rail i think we will have to change this a bit, right? maybe we can even change how we configure shipitscript so that we don't have to do this magic in worker.yml, eg. convert to list of instances

Reply:
also it looks to me - looking at the shipitscript code - that only one shipit instance is allowed in code, while data structure in config allows for more. it think we can simplify `ship_it_instances` to

```yaml
shipit_instance:
    scope: { "$eval: "TASKCLUSTER_SCOPE" }
    api_root_v2: { "$eval": "API_ROOT_V2" }
    timeout_in_seconds: 60
    taskcluster_client_id: { "$eval": "TASKCLUSTER_CLIENT_ID" }
    taskcluster_access_token: { "$eval": "TASKCLUSTER_ACCESS_TOKEN" }
```